presto-parser/src/main/java/com/facebook/presto/sql/parser/ParsingOptions.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.sql.parser;
+
+import static java.util.Objects.requireNonNull;
+
+public class ParsingOptions
```

Comment:
We already have a SqlParserOptions for this. We should add this there, instead.

Reply:
`SqlParserOptions` is a singleton, injected to `SqlParser` (also a singleton). And this one depends on the session. If we remove the session flag and leave only toggle in` FeatureConfig` then `ParsingOptions` and `SqlParserOptions` can be merged into one.

Reply:
We could make the SqlParserOptions that gets injected into SqlParser the "default" configuration for that instance, and whichever is passed for each invocation would override those defaults.

Reply:
Yeah, that's how I envisioned it working. But, I agree. This is temporary, so not worth it.